imap-proto/src/types.rs

```diff
@@ -925,6 +931,9 @@ impl<'a> QuotaRoot<'a> {
     }
 }
 
+pub mod acls;
```

Comment:
Style nit: I like to keep these closer to the top, in a separate block below the imports.

Reply:
fully agree with you there.   This was just due to refactoring while I was building it out as I originally had all the types in this file, but it was getting too large.